tools/azsdk-cli/Azure.Sdk.Tools.Cli/Helpers/Process/Options/ProcessOptions.cs

```diff
@@ -17,6 +17,8 @@ public interface IProcessOptions
 public class ProcessOptions : IProcessOptions
 {
     public static readonly int DEFAULT_PROCESS_TIMEOUT_SECONDS = 120;  // Default timeout of 2 minutes
+    private static readonly TimeSpan maxProcessTimeout = TimeSpan.FromHours(2);
```

Comment:
This is a fair timeout, I know in Java running tests in record mode can sometimes take over an hour. Just sharing a general thought as how this affects the user experience for such lengthy operations, but I assume they were aware of this when they made their decision.